modules/installation-and-upgrade/pages/troubleshooting.adoc

```diff
@@ -86,3 +86,25 @@ This duplication can confuse some browsers.
 
 This issue is resolved by clearing the cache and reloading the page.
 In most browsers, you can do this quickly by pressing kbd:[Ctrl+F5].
+
+
+
+== Registering a traditional client as Salt minion after deleting it
+
+This is not a valid scenario. Normally you migrate a traditional client to a Salt minion without deleting the client.
+Salt automatically detects that you have a traditional client and does the necessary changes itself. But if you somehow
+deleted the traditional client and want to register it as a Salt minion again, you have to do the following steps on the
+client before registering it as Salt minion:
```

Comment:
```suggestion
////
Salt automatically detects that you have a traditional client and does the necessary changes itself.
But if you somehow deleted the traditional client and want to register it as a Salt minion again, you have to do the following steps on the client before registering it as Salt minion:
////
```
I think we do not need the above text.  But keeping for reference for the moment, commented.